src/Tempest/Database/Drivers/MySqlDriver.php

```diff
@@ -37,4 +38,9 @@ public function getPassword(): ?string
     {
         return $this->password;
     }
+
+    public function createQueryStatement(string $table): QueryStatement
```

Comment:
create the `QueryStatement` class through the driver

Reply:
👍